src/templates/slideshow/slideshow.test.js

```diff
@@ -1,7 +1,7 @@
 describe('Make sure slideshow loads', () => {
   it('should have background-image', () => {
     cy.visit('http://localhost:3000');
-    cy.get('#debug-bar-fixture-7').click();
+    cy.get('select').select('slideshow2');
```

Comment:
It's better to use a class or ID as selector, since other 'select' elements might exist.